src/node/desktop/src/main/url-utils.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * url-utils.ts
+ *
+ * Copyright (C) 2022 by RStudio, PBC
```

Comment:
Dangit, you're going to make me update my in-progress PR changing all RStudio copyrights to Posit, aren't you?